.github/workflows/publish-docs.yml

```
@@ -59,7 +59,13 @@ jobs:
         run: yarn prettier
 
       - name: Run shellcheck
-        run: make shellcheck
+        uses: reviewdog/action-shellcheck@v1.13.0
+        with:
+          reporter: github-pr-review
+          filter_mode: nofilter
+          path: bin
```

```suggestion
          path: |
            bin
            .husky
```

Those should be scanned too 😉